src/Extension/BetterNavigatorExtension.php

```diff
@@ -111,9 +107,9 @@ private function generateNavigator()
                 }
             }
         }
-        
+
         // Only show edit link if user has CMS access
-        if($isDev || Permission::check('CMS_ACCESS_CMSMain')) {
+        if($isDev || Permission::check($this->owner->config()->better_navigator_edit_permission, $this->owner->config()->better_navigator_edit_permission_mode)) {
```

Comment:
Need a small fix here to prevent an undefined variable notice
```suggestion
        $editLink = null;
        if($isDev || Permission::check($this->owner->config()->better_navigator_edit_permission, $this->owner->config()->better_navigator_edit_permission_mode)) {
```

Reply:
I rebased to incorporate that merged pull